esx_service/cli/vmdkops_admin.xml

```diff
@@ -0,0 +1,571 @@
+<?xml version="1.0"?>
+<plugin
+    xmlns="http://www.vmware.com/Products/ESX/6.0/esxcli/">
+    <!-- esxcli extension syntax version -->
+    <version>1.0.0</version>
+    <namespaces>
+        <namespace path="vdvs">
```

Comment:
How about we use "container storage manager", under that, we can have "volume", "vmgroup", "policy" as subcommend.